rakelib/lib/pwsh.rb

```diff
@@ -0,0 +1,347 @@
+# frozen_string_literal: true
```

Comment:
I'm not sure if this file should be somewhere other than `rakelib/lib/pwsh.rb`...

Reply:
I think the 'right' way to do this is to put this in it's own rake task, and make that task a prerequisite for the two pwsh tasks.

```
task cmdlet_reference: :generate_pwsh_cmdlets do
...
```